dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <TheHeader v-if="!useMobileView" />
   <TheContent class="paper-section flex">
-    <CompanyInfoSheet :company-id="companyId" />
+    <CompanyInfoSheet :company-id="companyId" @fetched-company-information="getCompanyName" />
     <div class="card-wrapper">
       <div class="card-grid">
+        <ClaimOwnershipPanel v-if="!isUserDataOwner && isUserUploader && isCompanyIdValid" :company-id="companyId" />
```

Comment:
Does the company owner has to be a uploader? I thought a normal data_reader user could also claim ownership of a company and by this get uploader rights for this company

Reply:
Yes please check it. But i think otherwise this story would not make sense: https://jira.d-fine.dev/browse/DALA-3268

Reply:
Ah yes - makes sense. Still waiting for an answer from Andreas.

Reply:
Should i wait with closing this until you have gotten an answer?

Reply:
Closing this now